tools/syz-declextract/README.md

```diff
@@ -0,0 +1,41 @@
+# syz-declextract
+## Linux Kernel (For testing purposes)
+```
+export KERNEL=$PWD/linux-stable
+git clone \
+https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git $KERNEL
+cd $KERNEL
+make CC=clang defconfig # Having clang as the compiler is optional but removes
+                        # erros later on
+./scripts/config -e FTRACE_SYSCALLS
+make CC=clang olddefconfig
+make CC=clang -j`nproc` # kernel has to be built at least once for the script
+                        # to work
+./scripts/clang/gen_compile_commands.py
+```
+## LLVM Project
+```
+LLVM=$PWD/llvm-project
+git clone https://github.com/llvm/llvm-project.git $LLVM
+echo 'add_clang_executable(syz-declextract syz-declextract/syz-declextract.cpp)
+target_link_libraries(syz-declextract PRIVATE clangTooling)' >> \
+$LLVM/clang/CMakeLists.txt
```

Comment:
Moving this the previous line should make it more readable.

Reply:
Done!